app/src/main/java/dev/arkbuilders/arkmemo/ui/fragments/SettingFragmentsV2.kt

```diff
@@ -0,0 +1,29 @@
+package dev.arkbuilders.arkmemo.ui.fragments
+
+import android.os.Bundle
+import android.view.View
+import androidx.fragment.app.Fragment
+import by.kirich1409.viewbindingdelegate.viewBinding
+import dev.arkbuilders.arkmemo.R
+import dev.arkbuilders.arkmemo.databinding.FragmentEditNotesV2Binding
+import dev.arkbuilders.arkmemo.databinding.FragmentSettingsBinding
+import dev.arkbuilders.arkmemo.utils.gone
+import dev.arkbuilders.arkmemo.utils.visible
+
+open class SettingFragmentsV2: Fragment(R.layout.fragment_settings) {
```

Comment:
```suggestion 
open class SettingsFragmentV2: Fragment(R.layout.fragment.settings) { 
```

Reply:
@shuberttheI Sorry I don't get what's your point here. You added a space into the line, what difference does it make?

Reply:
My suggestion is, the class name should be ```SettingsFragmentV2``` not ```SettingFragmentsV2```

Reply:
@shuberttheI It's updated now.
Thanks for pointing this out.